wbooks-api/app/services/open_library.rb

```diff
@@ -0,0 +1,22 @@
+class OpenLibrary
+  def self.valid_isbn?(isbn)
+    (isbn.length == 10 || isbn.length == 13) && isbn.scan(/\D/).empty?
+  end
+
+  def self.search(isbn)
+    return {} unless valid_isbn?(isbn)
+
+    response = HTTParty.get("https://openlibrary.org/api/books?bibkeys=ISBN:#{isbn}"\
+                            '&format=json&jscmd=data').values
+
+    return {} if response.empty?
+
+    response = response[0]
+
+    { isbn: isbn,
```

Comment:
In the projects we usually create a class that represent the response. It's ok to leave it like this here